src/assets/ProjectsImg/ProjectsImg.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export default function ProjectsImg() {
```

Comment:
Could we name this asset DefaultProjectsImg ?  only due to this being the image we'll use if none provided for the project